src/lib/util.ts

```diff
@@ -35,3 +37,13 @@ export async function getInstallDate(): Promise<Date> {
     return installDate;
   }
 }
+
+export async function lastDonationDate(db): Promise<Date | null> {
+  let donations = await db.getDonations(1);
+  return donations.length > 0 ? donations[0].date : null;
+}
+
+export async function secondsSinceDonation(db): Promise<number> {
+  const lastTime = (await lastDonationDate(db)) || (await getInstallDate());
+  return moment().diff(new Date(lastTime)) / 1000;
```

Comment:
Maybe just keep it in milliseconds and rename function since the only place it's used at the moment converts it back to millis

Reply:
in mounted funciton in dashboard page it is used to get seconds. could easily be changed to millis there as well though